.tool-versions

```diff
@@ -1,2 +1,2 @@
-nodejs 18.20.3
-pnpm 9.9.0
\ No newline at end of file
+nodejs 20.19
```

Comment:
I'm not sure what does `.tool-versions` do? Local development? But let's keep things in sync.